### DISCUSSION TITLE
newbies/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#201] [Website Discussions]
# New to Final Cut Pro?

If you're brand new to Final Cut Pro for Mac or Final Cut Pro for iPad- this is the place for you!

https://fcp.cafe/newbies/

**Comment by devbenegal** [comment]
Thomas Grove Carter had recommended the Ripple Training course which I bought and highly recommend. 
Another great resource is Tyler Stalman’s video Getting Started with Final Cut Pro. 
This was especially great as he explains the fundamentals in a clear and simple way. 

Helped me when I wanted to transition from Avid Film / Media Composer and Final Cut 7 to FCP X.

**Reply by latenitefilms** [reply]
Awesome, thank you! I've added Tyler's video - it will appear shortly.

**Comment by Dark-Petey** [comment]
Excellent set of tutorials tailored for beginners! I particularly enjoyed Ripple Training, which includes media for enhanced learning. I highly recommend incorporating a tutorial by Israel Hyman (Izzy) into your resources. His YouTube tutorials are truly outstanding, featuring a well-paced and exceptionally explained approach. Don't miss out on them – there's three hours of free Final Cut training available!

https://www.youtube.com/watch?v=_jvyrnQpR1E&list=PLp5SAgbM8S87n1qvv34-PAU2zMHLlXyfb
